config/routes.rb

```diff
@@ -0,0 +1,6 @@
+Rails.application.routes.draw do
+  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+
+  # Defines the root path route ("/")
+  # root "articles#index"
+end
```

Comment:
- [ ] To have the API endpoint for returning a random greeting created, kindly create a route here. This will be what the frontend will use to fetch the greetings created.